OxfordBodleian/Juel-Jensen/BDLaethg29.xml

```diff
@@ -151,99 +154,116 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     <q xml:lang="en">MS Aethiop 8 <foreign xml:lang="gez">ልፋፈ፡ ጽድቅ፡</foreign> Lefāfe Sedeq
                                         3rd quarter C 20 <placeName ref="LOC3577Gondar">Gondar</placeName> <date when="1973">1973</date></q>
                                 </item>
-                                
+
                                 <item xml:id="e3">
                                     <desc>Space left empty for the name of the owner in supplications
                                         is unfilled throughout (e.g. <locus target="#6r #14v #25r"/>). All supplications contain the word
-                                        <foreign xml:lang="gez">ለአመትከ፡</foreign>, "your maid", indicating that the manuscript was produced for a 
+                                        <foreign xml:lang="gez">ለአመትከ፡</foreign>, "your maid", indicating that the manuscript was produced for a
                                         woman.
                                     </desc>
                                 </item>
-                                
+
                                 <item xml:id="e4">
                                     <desc>
                                         Erroneously omitted characters have been added above the line on <locus target="#4r #11r"/>.
                                     </desc>
                                 </item>
                             </list>
                         </additions>
-                        
-                     <!--   <bindingDesc>
-                            <binding xml:id="binding">
-                                <decoNote xml:id="b1"></decoNote>
-                                <decoNote xml:id="b2" type="bindingMaterial">
-                           <material key="wood"/>
-                        </decoNote>
-                            </binding>
-                        </bindingDesc>-->
-                        
+
+                    <bindingDesc>
+                     <binding contemporary="Ethiopian" notBefore="1950">
+                         <decoNote xml:id="b1" type="bindingMaterial">
+                           The book is bound in <material key="wood">wooden</material> boards.
+                         </decoNote>
+                         <decoNote xml:id="b2" type="Boards">
+                           The boards are flush with the bookblock and have shallow external bevels. The grain runs vertically.
+                           Both the external and internal surfaces of the boards are rough.
+                         </decoNote>
+                         <decoNote xml:id="b3" type="SewingStations">2
+                           Chainstitch sewing on two sewing stations (one pair). The sewing is carried out with an untwisted thread. In the centrefold of
+                           <locus target="q1"/>, between the sewing stations, is present a single thread lenght with a knot while in the rest of the quires
```

Comment:
"is present a single thread length with a knot" I find not so well formulated and don't understand well

Reply:
Mmm...you are right...I tried to use the possible minor number of words to describe the thing :D
In the first quire, between the sewing stations, the ends of two threads are knotted resulting in a single stitch/thread length/passage of thread...I don't know if it is clearer D:

Reply:
I find it clearer like this. "In the centrefold of the first quire, between the sewing stations, the ends of two threads are knotted together resulting in a single thread length, while in the rest of the quires the thread lengths are double."